packages/lit-dev-content/site/docs/ssr/overview.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Server-side rendering (SSR)
+eleventyNavigation:
+  key: 🧪 Overview
+  parent: Server-side rendering
+  order: 1
+---
+
+<details class="pre-release">
+  <summary> 🧪 Pre-release software: not for production.</summary>
+  
+  Lit labs packages are published to get feedback from the wider community. This code shouldn't be used in production, but we encourage you to try it out and [file issues](https://github.com/lit/lit/issues/new/choose) for any bugs you find. For general feedback, please use the GitHub [discussion](https://github.com/lit/lit/discussions).
```

Comment:
We need to style the details so they're clearly part of the note and not the page content. Some kind of box. I also think the note needs some styling to make it like an alert.

I really like the GitHub alert styles with a colorful left border, like
<img width="385" alt="FTNFsSlX0AE8AXq" src="https://user-images.githubusercontent.com/522948/182971851-02860e50-d88e-4c23-9a6b-be4a5e2054e1.png">

But until we update all of our alert boxes, we should stick to a style consistent with our existing boxes:


<img width="786" alt="Screen Shot 2022-08-04 at 4 24 47 PM" src="https://user-images.githubusercontent.com/522948/182971946-52c3fa82-101a-4924-88dd-d5c9ad6914f9.png">

We should use the same labs icon in all places.

Reply:
This change replaces the note with a summary/details element.  Not sure how a summary/details inside a note would work, although that's an interesting possibility.

Reply:
I like @aomarks's suggestion here https://github.com/lit/lit.dev/pull/880#discussion_r938324232 to shorten & soften the wording so we won't need to collapse and just use our aside styling with the flask icon.

Reply:
Agreed.